notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>If <code>Unnamed: 0</code> column doesn't contain any meaningful information, could we just drop it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>if we generate numbers on the fly then this is not required</p>